angularjs-portal-home/src/main/webapp/my-app/layout/widget/controllers.js

```diff
@@ -223,12 +223,22 @@ define(['angular'], function(angular){
           var successFn = function(result){
             $scope.loading = false;
             $scope.data = result.data;
+            
             if($scope.data.responseStatus != 200) {
               $scope.error = true;
-            } else if(!$scope.data.responseData
-              || !$scope.data.responseData.feed
-              || $scope.data.responseData.feed.entries.length == 0) {
-              $scope.isEmpty = true;
+              $scope.loading = false;
+            } else {
+            		if(!$scope.data.responseData
```

Comment:
We typically use 2 spaces instead of tabs, you can upload the google styleguide to your eclipse and it will magically do that: https://github.com/google/styleguide/blob/gh-pages/eclipse-java-google-style.xml